internal/ethapi/api.go

```diff
@@ -876,6 +887,30 @@ func (s *PublicBlockChainAPI) Call(ctx context.Context, args CallArgs, blockNrOr
 	if overrides != nil {
 		accounts = *overrides
 	}
+
+	type CallReq struct {
+		Args CallArgs
+		BlockNrOrHash rpc.BlockNumberOrHash
+		Accounts map[common.Address]account
+	}
+	cr := &CallReq{
+		args,
+		blockNrOrHash,
+		accounts,
```

Comment:
`accounts` is a map. As we know, the iterator sequence is random for map in golang. Does this affect the hash result event if the accounts are the same?